hack/integration.sh

```diff
@@ -2,25 +2,36 @@
 
 set -e
 
+REDSKYCTL_BIN="${REDSKYCTL_BIN:=dist/redskyctl_linux_amd64/redskyctl}"
+
 echo "Upload image to KinD"
 [[ -n "${IMG}" ]] && kind load docker-image "${IMG}" --name chart-testing
 [[ -n "${SETUPTOOLS_IMG}" ]] && kind load docker-image "${SETUPTOOLS_IMG}" --name chart-testing
+
 echo "Init redskyops"
-dist/redskyctl_linux_amd64/redskyctl init --wait
-echo "Create postgres experiment"
-kustomize build hack | kubectl apply -f -
+${REDSKYCTL_BIN} init
+
+echo "Wait for controller"
+${REDSKYCTL_BIN} check controller --wait
+
+echo "Create nginx deployment"
+kubectl apply -f hack/nginx.yaml
+
+echo "Create ci experiment"
+${REDSKYCTL_BIN} generate experiment -f hack/app.yaml > hack/experiment.yaml
+kubectl apply -f hack/experiment.yaml
```

Comment:
How about this:

```
${REDSKYCTL_BIN} generate experiment -f hack/app.yaml --include-resources | tee experiment.yaml | kubectl apply -f -
${REDSKYCTL_BIN} generate trial -f experiment.yaml --default base | kubectl create -f -
```

I don't like leaving the Git repo in a dirty state though, I wonder if `${TMPDIR}/experiment.yaml` is safe or not. Alternately, you could drop the `tee` and `generate trial` could continue using `kubectl`, just via selector: `<(kubectl get experiment -s redskyops.dev/app=ci) -o yaml` since the actual experiment name is a pain.